website/snippets/_sl-env-vars.md

```diff
@@ -0,0 +1 @@
+Use [Extended Attributes](/docs/dbt-cloud-environments#extended-attributes) and [Environment Variables](/docs/build/environment-variables) when connecting to the Semantic Layer. If you set a value directly in the Semantic Layer Credentials, it will have a higher priority than Extended Attributes. When using environment variables, the default value for the environment will be used.
```

Comment:
@aiguofer just double checking that the user would be adding this in their env settings and then adding `DBT_ENV_SECRET` in the SL credentials page?

Reply:
correct.

For example, using something like `{{env_var('DBT_WAREHOUSE')}}` to set the `warehouse` value in your SL credentials would work.

Similarly, if you set your `account` value in your connection using Extended Attributes with a value like `{{env_var('DBT_ACCOUNT')}}`, we will now be able to do that look up for the Extended Attributes AND the env var.